### PR TITLE
Azure

### PR DESCRIPTION
**Pull request by khancyr**
Fix some Azure issue with  Python3

### CODE REVIEW

**Comment by peterbarker** [comment]
Merged, thanks!

File: .azure/autotest_template.yml [hunk 1]
```diff
@@ -16,9 +16,9 @@ jobs:
     displayName: Initialize MAVLink submodule
   - script: git submodule update --recursive --init modules/uavcan
     displayName: 'Initialize UAVCAN submodule'
-  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python setup.py build install --user"
+  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python3 setup.py build install --user"
```
Comment (OXINARF) [inline .azure/autotest_template.yml:19]: This change is wrong, there is no python3 executable, which is why someone added the links you forced below. This is breaking the nightly tests.

File: .azure/autotest_template.yml [hunk 2]
```diff
@@ -16,9 +16,9 @@ jobs:
...
     displayName: 'Install pymavlink from submodule'
-  - script: C:\Cygwin\bin\bash --login -c "pip install MAVProxy"
+  - script: C:\Cygwin\bin\bash --login -c "pip3 install MAVProxy"
```
Comment (OXINARF) [inline .azure/autotest_template.yml:21]: Same.

**Review by OXINARF — COMMENTED** [review]
@khancyr Can you fix these things?

File: .azure/autotest_template.yml (hunk 1 above)
Comment (khancyr) [reply .azure/autotest_template.yml:19]: I was pretty sure that the log show that python3  was there... Anyways, I will revert that. Thanks

**Pull request merged**